ffi/wgpu-remote.h

```diff
@@ -1,72 +0,0 @@
-
```

Comment:
Looks like this was accidentally removed, maybe it wasn't regenerated during `make`? If you can't get this to be generated, we could just amend the commit to keep it

Reply:
Yeah, something doesn't work correctly with ffi creation using make, not sure why. I'll fix it and try it again